packages/invoices/src/InboundShipment/DetailView/SidePanel.tsx

```diff
@@ -0,0 +1,148 @@
+import React, { FC } from 'react';
```

Comment:
c+p..!

Reply:
Not sure if the detail panel will end up the same as for outbound - but really, doesn't matter right now. just a thought for later.. that we will need a review of the site at some point.